후보자 정보 출력 기능 추가

jihyun0616 opened the pull request:
페이지 별 후보자 정보 출력 기능을 구현했습니다. 현재 API 쿼리 파라미터인 roomId는 하드코딩되어 있으며, 세션 정보(?다른 방식이 있을까요)를 기반으로 현재 접속 중인 방의 roomId를 추출하여 파라미터로 전달하는 방식으로 추후 수정이 필요합니다.

room 도메인과 분리하여 candidate 패키지를 따로 만들어 model, repository, controller를 구현하였습니다.

jihyun0616 committed:
후보자 정보 출력 기능 추가

페이지 별 후보자 정보 출력 기능을 구현했습니다. 현재 API 쿼리 파라미터인 roomId는 하드코딩되어 있으며, 세션 정보를 기반으로 현재 접속 중인 방의 roomId를 추출하여 파라미터로 전달하는 방식으로 추후 수정이 필요합니다.

choi2h reviewed (commented):
candidateController, candidateService 모두 candidate 디렉토리로 이동해도 될 것 같습니다.
API에 room이 들어가긴 하지만 `/candidate?roomId=3` 이렇게 하면 룸 아이디도 분리되고 오로지 후보자 관련된 API이기 때문에 candidate 디렉토리로만 관리해도 되지 않을까 싶습니다!

jihyun0616 commented:
> candidateController, candidateService 모두 candidate 디렉토리로 이동해도 될 것 같습니다. API에 room이 들어가긴 하지만 `/candidate?roomId=3` 이렇게 하면 룸 아이디도 분리되고 오로지 후보자 관련된 API이기 때문에 candidate 디렉토리로만 관리해도 되지 않을까 싶습니다!

확인하였습니다!

[기존]                                                            
room.service               
 ㄴ candidateService                                      

room.Controller                                               
 ㄴ candidateController                                



[수정]
**candidate.service**
 ㄴ candidateService

**candidate.controller**
 ㄴ candidateController


1. candidate service, controller 패키지를 별도로 만들어 room과 분리하여 관리
2.  /candidate?roomId=3로 API 수정
2-1. 이후 update, delete도 /candidate 아래로 관리

수정사항 반영하여 새롭게 request 하도록 하겠습니다!

jihyun0616 committed:
후보자 정보 출력 리팩토링

1. candidateController, candidateService를 모두 candidate 디렉토리로 이동시켰습니다.
2. api를 수정하였습니다.

his8457 reviewed (approved):
roomId 는 방생성화면에서 넘어올 때, entryCode(입장코드)와 함께 전달 될 거에요~!
